### PR TITLE
rename debug

### PR DESCRIPTION
**Pull request by brrmorre**
**Related issue**: 

**Type of change**: bug report | feature request | other enhancement

**Impact**: no functional change | API addition (no impact on existing code) | API modification

**Development Phase**: proposal |  implementation

**Release Notes**

### CODE REVIEW

File: src/main/scala/devices/debug/Periphery.scala [hunk 1]
```diff
@@ -132,19 +132,14 @@ trait HasPeripheryDebugModuleImp extends LazyModuleImp {
         c:= io.clock
         r:= io.reset
     }
-
```
Comment (brrmorre) [inline src/main/scala/devices/debug/Periphery.scala:135]: unnecessary formatting changes I should get rid of these

File: src/main/scala/devices/debug/Periphery.scala [hunk 2]
```diff
@@ -120,9 +120,14 @@ trait HasPeripheryDebugModuleImp extends LazyModuleImp {
 
     require(!(debug.systemjtag.isDefined && debug.apb.isDefined),
       "You cannot have both APB and JTAG interface in HasPeripheryDebugModuleImp")
+  
+    debug
+  }
 
-    debug.clockeddmi.foreach { dbg => outerdebug.module.io.dmi.get <> dbg }
-
+  debug.foreach { debug =>
```
Comment (jackkoenig) [inline src/main/scala/devices/debug/Periphery.scala:127]:
```suggestion
  // Workaround for https://github.com/freechipsproject/chisel3/issues/1603
  debug.foreach { debug =>
```

File: src/main/scala/devices/debug/Periphery.scala [hunk 3]
```diff
@@ -120,9 +120,14 @@ trait HasPeripheryDebugModuleImp extends LazyModuleImp {
...
+    debug.clockeddmi.foreach { 
+    val outerdebug = outer.debugOpt.get
+    dbg => outerdebug.module.io.dmi.get <> dbg }
```
Comment (jackkoenig) [inline src/main/scala/devices/debug/Periphery.scala:130]:
```suggestion
    val outerdebug = outer.debugOpt.get
    debug.clockeddmi.foreach { dbg => outerdebug.module.io.dmi.get <> dbg }
```

Comment (jackkoenig) [reply src/main/scala/devices/debug/Periphery.scala:130]: `outerdebug` is needed below

**Review by hcook — APPROVED** [review]
